docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -41,27 +41,57 @@ Example task:
 +
 [source]
 --
-name: example-task
-      params:
-      - name: example-param
-        value: “Example”
-      runAfter:
-      - build-container #You can be more specific by choosing another task
-      taskRef:
-        params:
-        - name: name
-          value: example-task
-        - name: bundle
-          value: <path to container image>
-        - name: kind
-          value: task
-        resolver: bundles
-      when:
-      - input: $(params.skip-checks)    #This references the pipeline parameters
-        operator: in
-        values:
-        - "false"
+  name: example-task
+  params:
+  - name: example-param
+    value: “Example”
+  runAfter:
+  - build-container #You can be more specific by choosing another task
+  taskRef:
+    params:
+    - name: name
+      value: example-task # metadata.name field of the Task
+    - name: bundle
+      value: <full reference url to the tekton bundle> # For more details on tekton bundles, refer https://tekton.dev/docs/pipelines/pipelines/#tekton-bundles
```

Comment:
Please put this comment in a single line.

Reply:
Done.